package.json

```diff
@@ -78,20 +78,21 @@
     "uiFramework:compileCss": "cd packages/osd-ui-framework && yarn compileCss",
     "osd:watch": "scripts/use_node scripts/opensearch_dashboards --dev --logging.json=false",
     "build:types": "scripts/use_node scripts/remove.js ./target/types && tsc --p tsconfig.types.json",
-    "docs:acceptApiChanges": "scripts/use_node --max-old-space-size=6144 scripts/check_published_api_changes.js --accept",
+    "docs:acceptApiChanges": "scripts/use_node --max-semi-space-size=16 --max-old-space-size=6144 scripts/check_published_api_changes.js --accept",
     "docs:generateDevDocs": "scripts/use_node scripts/generate_docs_sidebar.js",
     "osd:bootstrap": "scripts/use_node scripts/build_ts_refs && scripts/use_node scripts/register_git_hook",
     "spec_to_console": "scripts/use_node scripts/spec_to_console",
     "pkg-version": "scripts/use_node -e \"console.log(require('./package.json').version)\"",
     "release_note:generate": "scripts/use_node scripts/generate_release_note",
     "cypress:run-without-security": "env TZ=America/Los_Angeles NO_COLOR=1 cypress run --env SECURITY_ENABLED=false",
     "cypress:run-with-security": "env TZ=America/Los_Angeles NO_COLOR=1 cypress run --env SECURITY_ENABLED=true,openSearchUrl=https://localhost:9200,WAIT_FOR_LOADER_BUFFER_MS=500",
-    "osd:ciGroup10": "echo \"cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/01/*.spec.js\"",
+    "osd:ciGroup10": "echo \"cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/01/*.spec.js,!cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/01/saved_search.spec.js\"",
     "osd:ciGroup11": "echo \"cypress/integration/dashboard_sanity_test.spec.ts\"",
     "osd:ciGroup12": "echo \"cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/02/*.spec.js\"",
     "osd:ciGroup13": "echo \"cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/03/*.spec.js\"",
     "osd:ciGroup14": "echo \"cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/04/*.spec.js\"",
     "osd:ciGroup15": "echo \"cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/05/*.spec.js\"",
+    "osd:ciGroup16": "echo \"cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/01/saved_search.spec.js\"",
```

Comment:
is this and above change for `osd:ciGroup10` a temporary thing that you will revert, or you think it should be permanent?